assets/js/Viewer.jsx

```diff
@@ -18,16 +18,28 @@ class Viewer extends Component {
       <div>
         {this.sortedKeys().map((key) => {
```

Comment:
I wouldn't bother with this class method `sortedKeys`. I'd just add the `const { signs } = this.props` at the top of the render method and call `Object.keys(signs).sort().map(key => ...` here, or `const sortedKeys = Object.keys(signs).sort()` and `sortedKeys.map(key => ...`

Reply:
sounds good to me

Reply:
i ended up getting linter complaints about this not being stateless, so i will leave it as is for now since its not really being touched by this pr anyway.

Reply:
I was baffled by this so I changed it locally and it passes lint, so I'm just going to push the baby commit with this change